pkg/arvo/sys/hoon.hoon

```diff
@@ -6654,6 +6669,7 @@
     {$yell p/(list hoon)}                               ::  render as tank
     {$xray p/manx:hoot}                                 ::  ;foo; templating
   ::                                            ::::::  cores
+    [%brbs sample=spec body=spec]
```

Comment:
Please use the column 57 `::` and old-style `{}/` for consistency with surrounding code.  I'd be happy with a separate commit that changed it for all of `+hoon`, but I don't like mixing styles within a single type definition.

Reply:
Funny, I remember saying this exact same thing to someone. *stares at @frodwith *

Reply:
Also, yes I'll change this, but only to make you happy, and only because I'm going to be changing all of this Soon (tm) anyway.